src/kata5.reachDestination.js

```diff
@@ -1,3 +1,13 @@
-const reachDestination = (distance, speed) => {};
+const reachDestination = (distance, speed) => {
+    const time = Math.round(2 * distance / speed) / 2;
```

Comment:
The same as above, logic is perfect, but it can be refactored by using the ternary operator :)